src/components/PluginIssues.jsx

```diff
@@ -0,0 +1,70 @@
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+
+function PluginIssues({pluginId}) {
+    const [isLoading, setIsLoading] = useState(false);
+    const [issues, setIssues] = useState([]);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            setIsLoading(true);
+            const result = await axios(`/api/plugin/${pluginId}/issues/open`);
+            setIssues(result.data.issues || []);
+            setIsLoading(false);
+        };
+        fetchData();
+        return;
+    }, []);
+
+    if (isLoading) {
+        return (<div className="spinner-border" role="status">
+            <span className="sr-only">Loading...</span>
+        </div>);
+    }
+
+    return (
+        <div>
+            <div className="table-responsive">
+                <table className="table">
+                    <caption>List of issues</caption>
+                    <thead>
+                        <tr>
+                            <th scope="col">Key</th>
+                            <th scope="col">Summary</th>
+                            <th scope="col">Assignee</th>
+                            <th scope="col">Reporter</th>
+                            <th scope="col">Priority</th>
+                            <th scope="col">Status</th>
+                            <th scope="col">Resolution</th>
```

Comment:
These columns are quite specific to jira, is it worth abstracting this into a `JiraPluginIssues`?

Reply:
Phase 2 yes, though the abstraction would be on API not ui

Reply:
I would assume UI would need to handle it and delegate to the correct UI though? but yes later sounds good